tests/e2e/specs/conference.spec.js

```diff
@@ -0,0 +1,28 @@
+// about cypress and iframes https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/
+const { expect } = require('chai');
+
+const getIframeDocument = () => cy
+  .get('#jitsiConferenceFrame0')
+  .its('0.contentDocument')
+  .should('exist');
+
+const getIframeBody = () => getIframeDocument()
+  .its('body').should('not.be.undefined')
+  .then(cy.wrap);
```

Comment:
For me, it is unclear how it works

Reply:
@davidyuk 
I can rewrite as .then(iframeBody => cy.wrap(iframeBody));

Reply:
maybe you mean about the cy.wrap method itself?

Reply:
yes, I can't get why do you need that at all, here is a wrap doc for reference: https://docs.cypress.io/api/commands/wrap.html#Syntax